src/app/portlets/content-types/fields/content-type-fields-properties-form/content-type-fields-properties-form.component.ts

```diff
@@ -189,13 +162,24 @@ export class ContentTypeFieldsPropertiesFormComponent extends BaseComponent impl
         }
 
         this.handleDisabledRequired(propertyValue);
+        this.handleDisabledIndexed(true);
     }
 
     private handleDisabledIndexed(disable: boolean): void {
-        disable ? this.form.get('indexed').disable() : this.form.get('indexed').enable();
+        if (this.form.get('indexed')) {
+            disable ? this.form.get('indexed').disable() : this.form.get('indexed').enable();
+        }
     }
 
     private handleDisabledRequired(disable: boolean): void {
-        disable ? this.form.get('required').disable() : this.form.get('required').enable();
+        if (this.form.get('required')) {
```

Comment:
I think this should add a new test

Reply:
If we had tests... I think we should refactor the way we do that thing of the checkboxes, but we don't have time right now.

Reply:
but I think we should create the new test anyway

Reply:
Yep, you're righ, I'm on it